fields.f90

```diff
@@ -4391,9 +4391,11 @@ subroutine FLReadInput(NPTorder,Npolyads,Natoms,Nmodes,Jrot)
      !    
    endif
    !
-   if (trim(trove%symmetry)=='C2VN'.and.sym%N<job%bset(0)%range(2)) then
-      write (out,"('FLinput: The C2VN number',i5,' must be defined and equal to (or <) krot',i5)") sym%N,job%bset(0)%range(2)
-      stop 'FLinput - The C2VN number is undefined or too small'
+   if (trim(trove%symmetry)=='C2VN') then
+     if (sym%N<job%bset(0)%range(2)) then
```

Comment:
Is this just for readability? Does the `.and.` not shortcircuit?

Reply:
This fixes a segfault so I'm guessing either `sym%N` or `job%bset(0)%range(2)` are only allocated when `trove%symmetry)=='C2VN'`.

Reply:
I assumed it wouldn't matter but apparently [short-circuting is not standard](https://github.com/j3-fortran/fortran_proposals/issues/19)! (even though gfortran with optimisations supports it, for instance)